libgeopm/src/Profile.cpp

```diff
@@ -87,6 +86,10 @@ namespace geopm
             return;
         }
         connect();
+#ifndef GEOPM_DEBUG
+        // When debug not enabled, use of m_region_ids is optimized out
```

Comment:
Isn't it obvious that m_region_ids is optimized out looking at where/how it's used (i.e. all the GEOPM_DEBUG checks)?

I think what's not obvious here is why you have to manually insert HASH_UNMARKED (because we never call (create) region() on it).

Reply:
This is to avoid clang warnings about unused member variables.